runelite-api/src/main/java/net/runelite/api/ObjectID.java

```diff
@@ -28985,11 +28985,11 @@
 	public static final int NULL_28997 = 28997;
 	public static final int NULL_28998 = 28998;
 	public static final int NULL_28999 = 28999;
-	public static final int NULL_29000 = 29000;
+	public static final int STASH_MEDIUM_29000 = 29000;
```

Comment:
This file is auto generated. Do not rename fields in this class as they will be overridden when it is updated.